src/attributes/attr.js

```diff
@@ -114,19 +114,21 @@ boolHook = {
 };
 
 jQuery.each( jQuery.expr.match.bool.source.match( /\w+/g ), function( i, name ) {
-	var getter = attrHandle[ name ] || jQuery.find.attr;
+	var getter = attrHandle[ name.toLowerCase() ] || jQuery.find.attr;
+
+	attrHandle[ name.toLowerCase() ] = function( elem, name, isXML ) {
```

Comment:
This is just looping over the attrs in the regex. Is it necessary to lowercase here?